ptxboa/data/2030_high.csv

```diff
@@ -9,14 +9,13 @@ key,parameter_code,process_code,flow_code,source_region_code,target_country_code
 "CALOR,,LOHC-L,,",CALOR,,LOHC-L,,,33.33,kWh H2 (l)/kg,Linde
 "CALOR,,NH3-L,,",CALOR,,NH3-L,,,5.199999999999999,kWh NH3 (l)/kg,Wikipedia with conversions
 "CAPEX,AEL-EL,,,",CAPEX,AEL-EL,,,,1210.9576551357125,USD2023/kW,
-"CAPEX,ATR,,,",CAPEX,ATR,,,,895.7895675646487,USD2023/kW,Oni et al. 2022
```

Comment:
Looks like all parameters for ATR and REGASATR (methane reconversion with and without regasification) have been removed. I suppose this was not intentional, as we still use these processes. I will check with Roman.